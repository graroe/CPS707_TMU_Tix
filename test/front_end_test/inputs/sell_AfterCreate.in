login
admin
create
Lecture1
20231028
350
sell
Lecture1
logout