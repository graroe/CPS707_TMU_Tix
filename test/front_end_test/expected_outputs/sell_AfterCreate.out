Enter session type <sales or admin>: 
Admin login successful.
Enter new event name <max 15 chars>: 
Enter date of event <format YYYYMMDD>: 
Enter number of available tickets: 
Event Lecture1 created.
Enter event name: 
Event not available for sale yet.
Session terminated.